### PR TITLE
Move tests, upgrade near-sdk, borsh

### PR DESCRIPTION
**Pull request by evgenykuzyakov**
Required for importing nearcore runtime standalone and write integration test

### CODE REVIEW

**Comment by ailisp** [comment]
cancel build as #85 introduce a regression cause ci stuck forever

File: libs-rs/rust-toolchain [hunk 1]
```diff
@@ -0,0 +1 @@
+nightly-2020-05-15
```
Comment (nearmax) [inline libs-rs/rust-toolchain:1]: Our contracts should be compilable with stable. What would be the reason to use nightly here?

Comment (ailisp) [reply libs-rs/rust-toolchain:1]: For intention to run cross contract test:
```
near-runtime-standalone = { git = "https://github.com/nearprotocol/nearcore.git" }
```
and nearcore need nightly to build. Maybe we should
- specify stable-xxx in rust-toolchain to produce determinstic build on contracts
- use script to switch to nightly to build cross contract test

Comment (ailisp) [reply libs-rs/rust-toolchain:1]: Oh actually @evgenykuzyakov added
```
RUSTFLAGS='-C link-arg=-s' cargo +stable build --target wasm32-unknown-unknown --release
```
in build.sh, this is how we did in core-contracts, so contracts are built with stable. but problem is we can't specify a determined rust version here

**Comment by ailisp** [comment]
Ping @nearmax ptal :)

**Pull request merged**